function/index.js

```diff
@@ -55,11 +56,10 @@ async function handler(event, context) {
     const response = await sqsService.receiveSQS(receiveInput);
 
     // Return early if there are no messages to consume.
-    if (!response.Messages) {
+    if (response?.Messages === undefined || response?.Messages === null) {
```

Comment:
I think `!response?.Messages` should check it's not undefined or null by default